app/routes/circulars/circulars.server.ts

```diff
@@ -110,6 +110,62 @@ export async function search({
 
   const [startTime, endTime] = getValidDates(startDate, endDate)
 
+  const esQuery = query
+    ? {
+        function_score: {
+          query: {
+            bool: {
+              must: [
+                {
+                  query_string: {
```

Comment:
According to https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-string-query.html#query-string-syntax, `query_string` activates Lucene query syntax, like `author:"John Smith"`. That's probably a very good thing, but the UI should explain the supported syntax. Also, do we want Lucene syntax always on?

Reply:
If it were not always on, on what criteria would we determine when it should be applied?

Reply:
would this be sufficient?
const isLuceneSyntax = query.includes(':')

Reply:
I did change this, but leaving this comment open to make sure that the check I put in place is acceptable. I ran three tests:
1. I used lucene syntax
2. I used a search phrase with multiple words that occurred together
3. I used multiple search terms that did not appear in the same place in the text body

Reply:
> would this be sufficient? const isLuceneSyntax = query.includes(':')

Well, those wildcards wouldn't do anything if the user employs Lucene syntax. And they still won't do what you intend (which is match partial words) if the user inputs multiple terms separated by spaces.

Does OpenSearch have any query mode that does partial word matches?

Reply:
https://opensearch.org/docs/2.1/opensearch/ux/
https://opensearch.org/docs/latest/query-dsl/term/fuzzy/

Reply:
As I understand the documentation on [Lucene query syntax](https://opensearch.org/docs/latest/query-dsl/full-text/index/#query-string), you don't need wildcards in order to perform fuzzy matches. You can do that just with the [fuzzy query options](https://opensearch.org/docs/latest/query-dsl/full-text/index/#fuzzy-query-options).